coog/templates/customer-backend/deployment.yaml

```diff
@@ -38,8 +41,14 @@ spec:
             procMount: Default
             runAsNonRoot: true
             runAsUser: 1000
+          resources:
+            {{- toYaml .Values.coog.initContainers.resources | nindent 12 }}
+          command: ['yarn', 'build', '--no-optimization']
+          volumeMounts:
+            - name: builddir
+              mountPath: "/workdir/build"
           env:
-            - name:  BACKEND_PORT
+            - name:  PORT
```

Comment:
What is this port ? If this is the application port, can't we set it in dockerfile only ?

Reply:
Port of backend service.
I don't know, if, somehow, someday, someone wants to change it?

Reply:
We don't really care because on Kubernetes this is only the port for the PODs that we can easily change in service parameters and on Docker we can do the samething

Reply:
For more information, we are currently linking the HTTP port by default to the PODs port in this configuration : 

https://github.com/coopengo/coog-helm-charts/blob/master/coog/templates/customer-frontend/service.yaml